talent/contract.md

```diff
@@ -1,19 +1,25 @@
+
+You can find here the [English](https://drive.google.com/file/d/1LMKOmpTeLQZlEEKK7CkiJW8iC7HC-I0O/view?usp=sharing) and [Spanish](https://drive.google.com/file/d/1bDNPwht2GVnz8p3AfRDDp_mOxLFgetXr/view?usp=sharing)  version of our contract.
+
 ### Main clauses
-1. Description of your position and where you will work. Even though it says you'll work from our Madrid office, we have a <a href="">remote working policy</a> that allows you to work from home as well. <b>link missing</b>
+1. Description of your position and where you will work. Even though it says you'll work from our Madrid office, we have a remote working policy that allows you to work from home as well. We have an additional clause explaining it. 
 2. x
-3. In Spain, the weekly working hours need to be established for full-time contracts at 40h/week. We have this clause to comply with the law, but actually offer a <a href="">flexible working schedule</a> where you decide how many hours to work. As long as stuff gets done, we don't care if it takes 1h or 40h <b>link missing</b>
+3. In Spain, the weekly working hours need to be established for full-time contracts at 40h/week. We have this clause to comply with the law, but actually offer a [flexible working schedule](https://github.com/src-d/guide/blob/master/talent/flexible_holidays_working_schedule_remote_work.md) where you decide how many hours to work. As long as stuff gets done, we don't care if it takes 1h or 40h
 4. The contract hs an indefinite duration and the Spanish standard of 6 trial months. <b>This means that...</b>
 5. This clause details how much your gross salary will be.
-6. At source{d}, we offer 23 day of holidays a year. We have a <a href="">holiday policy</a> by which you can work on bank holidays or weekends though and exchange it for more days. Also, any unused holidays will be carried over to the next year. <b>link missing</b>
-7. x
-8. The legislation of the statute of workers approved by the <a href="https://www.boe.es/buscar/doc.php?id=BOE-A-1995-7730">Royal Legislative Decree 1/1995 of 24th March</a> and the collective agreement of <a href="https://www.boe.es/boe/dias/2009/04/04/pdfs/BOE-A-2009-5688.pdf"><k>Empresas de Consultoría y Estudios Opinión Pública</k></a> will be applied to this contract.
+6. At source{d}, we offer 23 day of holidays a year. We have a holiday policy by which you can work on bank holidays or weekends though and exchange it for more days. Also, any unused holidays will be carried over to the next year. Also, we have an additional clause including more details.
+7. The legislation of the statute of workers approved by the <a href="https://www.boe.es/buscar/doc.php?id=BOE-A-1995-7730">Royal Legislative Decree 1/1995 of 24th March</a> and the collective agreement of <a href="https://www.boe.es/boe/dias/2009/04/04/pdfs/BOE-A-2009-5688.pdf"><k>Empresas de Consultoría y Estudios Opinión Pública</k></a> will be applied to this contract.
+8. x
```

Comment:
It use to be used `n/a` for [not applicable items](https://en.wikipedia.org/wiki/n/a)